UPGRADING.rst

```diff
@@ -1,5 +1,57 @@
-Upgrading to Cerberus 1.0
-=========================
+Upgrading to Cerberus 2.0 from any 1.x version
+==============================================
+
+Major Changes and Deprecations
+------------------------------
+
+Rules
+.....
+
+A few rules have been renamed:
+
+========================  ===============
+Old name                  New name
+========================  ===============
+``keyschema``             ``keysrules``
+``schema`` for sequences  ``itemsrules``
```

Comment:
`schema` was a better choice for this, IMO.

Reply:
please discuss this in #385. in any case: we need to clearly distinguish the currently two `schema` rules.